### ISSUE TITLE
Comparison with growth form

### ISSUE THREAD

**Issue by wcornwell** [traitecoevo/australian_plant_endemism#7]
Linking results to growth form from `austraits` and making a figure

**Comment by wcornwell** [comment]
https://traitecoevo.github.io/austraits/

**Comment by wcornwell** [comment]
Two of these: one for endemic and one for non-endemic: <img width="1002" alt="Screen Shot 2022-06-22 at 10 16 43 am" src="https://user-images.githubusercontent.com/5534161/174917115-c041093f-f630-4ad6-8175-4febb6d511c3.png">

**Comment by wcornwell** [comment]
possible tutorial: https://r-graph-gallery.com/treemap.html

**Comment by stephwoolley** [comment]
Initial attempt at the endemic growth form (the larger groupings) treemap, i'm still trying to fix the colours though so it is a bit brighter. 
![Screen Shot 2022-07-08 at 12 19 38 pm](https://user-images.githubusercontent.com/106574502/177905033-8e8ae6be-2b95-419a-8ce0-54e809ab3fad.png)

**Comment by wcornwell** [comment]
have you tried to split the endemic and non-endemic species yet?  (if not use the updated data here: https://github.com/traitecoevo/australian_plant_endemism/blob/master/intermediate_data/country_dataset.csv )

**Comment by stephwoolley** [comment]
yep! i've done it with the initial growth form values, i've still got to recode them so they are in larger groups 
<img width="764" alt="Screen Shot 2022-07-08 at 1 58 52 pm" src="https://user-images.githubusercontent.com/106574502/177914362-5f470bf1-d894-4477-a833-4580bdfd0851.png">
<img width="768" alt="Screen Shot 2022-07-08 at 1 58 34 pm" src="https://user-images.githubusercontent.com/106574502/177914365-c8bc63ae-70e2-4b79-83dd-b086cb3f3f77.png">

**Comment by wcornwell** [comment]
interesting, so shrubs are more likely to be endemic than the average species?

**Comment by stephwoolley** [comment]
yep it looks like it, it should be more noticeable as well once i've joined all the sub-shrub groups!
